user/noto/Frontend.rst

```diff
@@ -39,10 +39,14 @@ The calibration mark can be manually switched on and off respectively with::
     > calOn 
     > calOff
 
+.. admonition:: WARNING:  
+
+    * Even if the calibration diode is a frontend device, its control is indirectly done through the 
+      Total Power so the "calmux" should be properly configured (check more details on backend section).
```

Comment:
Even if the calibration diode is a device included in the frontend, it is controlled through the TotalPower backend. As a consequente, the "calmux" must properly be configured - see details in the backend section.